workspaces/tech-radar/packages/backend/src/modules/auth/github.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Added this modules/auth section to the backend package to facilitate testing tech radar backend with real Github URls/authentication. Direct copy / paste from existing gh auth setup in [github-actions plugins in this repo](https://github.com/backstage/community-plugins/tree/main/workspaces/github-actions/packages/backend/src/modules/auth)

Reply:
No, it's to enable testing with a GH URL that requires authentication. In order to do that, you need to be logged into the Backstage instance with GH credentials (which URLReader service will then utilize when reading the URL you provide)

Reply:
mmm I don't think this is needed as the URLReader service uses a token stored in app-config for fetching the data.
I think if you use the guest provider it should work as well

Reply:
ah you're right! I misunderstood where URlReader grabbed its auth from - removing this in favor of guest provider then.